cfme/configure/tasks.py

```diff
@@ -11,13 +11,14 @@
 
 from cfme import web_ui as ui
 from cfme.base.login import BaseLoggedInPage
-from cfme.web_ui import toolbar as tb
 import cfme.fixtures.pytest_selenium as sel
 from cfme.web_ui import Form, Region, CheckboxTable, fill, match_location
 from cfme.utils.appliance import Navigatable
 from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
 from cfme.utils.log import logger
 from cfme.utils.wait import wait_for, TimedOutError
+import re
```

Comment:
**REQUIRED**
Please follow the style guides http://cfme-tests.readthedocs.org/guides/dev_guide.html